arangod/Sharding/ShardingInfo.h

```diff
@@ -55,7 +55,7 @@ class ShardingInfo {
   std::string shardingStrategyName() const;
 
   LogicalCollection* collection() const;
-  void toVelocyPack(arangodb::velocypack::Builder& result, bool translateCids) const;
```

Comment:
Please check in C++ file, if we can get rid of this input parameter again.
This is rather confusing for my Brain.

Reply:
Why? Do not fully understand. This is current devel-state.

Reply:
Aha only seen the red line not the green one. Gut. Removed parameter.